python_modules/libraries/dagster-aws/dagster_aws/s3/resources.py

```diff
@@ -91,6 +96,7 @@ def example_s3_solid(context):
         region_name=context.resource_config.get("region_name"),
         endpoint_url=context.resource_config.get("endpoint_url"),
         use_unsigned_session=context.resource_config["use_unsigned_session"],
+        profile_name=context.resource_config.get("profile_name"),
```

Comment:
Can you also update the documentation a few lines up to include this new optional config?